File: src/irrgarten/Juego.java
package irrgarten;
import irrgarten.controller.*;
import irrgarten.UI.*;

public class Juego {
    public static void main(String args[]){
        Game game = new Game(2);
        JFrame vista = new JFrame();
        Controller control = new Controller(game, vista);
        control.play();
    }
    
}

File: JUEGO-RUBY/irrgarten/juego.rb
require_relative 'Weapon'
require_relative 'Shield'
require_relative 'Dice'
require_relative 'GameState'
require_relative 'Monster'
require_relative 'Player'
require_relative 'Labyrinth'
require_relative 'Game'
require_relative 'Controller/controller'
require_relative 'UI/textUI'

juego = Irrgarten::Game.new(2)
vista = UI::TextUI.new
controlador = Control::Controller.new(juego,vista)
controlador.play